react/week1/todo-list/src/App.js

```diff
@@ -0,0 +1,28 @@
+import logo from "./logo.svg";
+import "./App.css";
+const GetToDo = (props) => {
+  return (
+    <div className="todos">
+      <h4>{props.description}</h4>
+      <p> {props.deadline} </p>
+    </div>
+  );
+};
+
+const todoList = [
+  { description: "Get out of bed", deadline: "Wed Sep 13 2017" },
+  { description: "Brush teeth", deadline: "Thu Sep 14 2017" },
+  { description: "Eat breakfast", deadline: "Fri Sep 15 2017" },
+];
+function App() {
+  return (
+    <div className="App">
+      <h2>My to do list</h2>
+      {todoList.map((x) => {
+        return <GetToDo description={x.description} deadline={x.deadline} />;
+      })}
```

Comment:
Just a small comment:
The console is showing a warning that says `Each child in a list should have a unique "key" prop.`

you can avoid the problem by simply providing a uniqe `key` that have the value of `todo.id` (you also have to add a `id` field for each todo in your `todoList`) 

```suggestion
      {todoList.map((x) => {
        return <GetToDo key={x.id} description={x.description} deadline={x.deadline} />;
      })}
```

You will learn more about it in the next weeks :)

Reply:
So thanks for great comment.